src/components/button/button_empty/index.ts

```diff
@@ -6,5 +6,9 @@
  * Side Public License, v 1.
  */
 
-export type { EuiButtonEmptyProps, EuiButtonEmptySizes } from './button_empty';
+export type {
+  EuiButtonEmptyProps,
+  EuiButtonEmptySizes,
+  EuiButtonEmptyPropsForButton,
```

Comment:
I know I'm being super annoying about this, but I don't _think_ we need to export this at the top level. For extremely specific types, we can/should reach directly into the file for the type and consumers can do the same as well (by pointing at our `src/` directories.


```suggestion
export type { EuiButtonEmptyProps, EuiButtonEmptySizes } from './button_empty';
  EuiButtonEmptyPropsForButton,
```

Happy to be argued against if others have arguments in favor of exporting this type publicly.